package.json

```diff
@@ -38,7 +38,7 @@
     "install:sass": "sass ../../src/style.scss ../../src/style.css --load-path ../../node_modules/@govflanders",
     "install:utils": "npm run install:bamboo && npm run install:copy:install && npm run install:copy:codeowner && npm run install:copy:github",
     "release": "np",
-    "release-component:prepare": "node build",
+    "release-component:prepare": "node ./scripts/checkDependencies.js && node ./scripts/build.js",
```

Comment:
Misschien dan ook best `exploder.js` en `generate-component-map.js` verplaatsen naar scripts.

Reply:
Ik heb dit bewust niet gedaan om te voorkomen dat indien er problemem zijn, het niet duidelijk is of het checkDependencies script de schuldige is, of het verplaatsen van de andere scripts.